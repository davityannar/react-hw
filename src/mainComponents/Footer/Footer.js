import React, {Component} from 'react';

import './Footer.scss';

export class Footer extends Component {
    render() {
        return (
            <div className='footer'>
                <div className='footer-wrapper'>
                    <p className='netflix'>
                      netflixroulette
                    </p>
                </div>
                
            </div>
        )
    }
}

export default Footer;



 


