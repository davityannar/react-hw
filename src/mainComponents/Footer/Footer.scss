.footer {
    display: flex;
    align-items: center;
    min-height: 40px;
    background-color: #414141;
  }
  
  .footer-wrapper {
    max-width: 1024px;
    width: 100%;
    margin: 0 auto;
  }

  .netflix {
    margin: 25px 0;
    padding-top: 50px;
    color: #f55263;
}