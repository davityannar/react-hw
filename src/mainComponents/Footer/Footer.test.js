import React from 'react'
import Footer from './Footer'

describe('Tests Footer container', () => {
  it('Matching snapshot', () => {
    const container = shallow(<Footer />)

    expect(container).toMatchSnapshot();
  })
})