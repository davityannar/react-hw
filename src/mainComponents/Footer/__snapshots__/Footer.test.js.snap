// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Footer container Matching snapshot 1`] = `
<div
  className="footer"
>
  <div
    className="footer-wrapper"
  >
    <p
      className="netflix"
    >
      netflixroulette
    </p>
  </div>
</div>
`;
